torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -253,6 +253,15 @@ def _is_int8_gemm(inputs):
         def reorder_and_filter(inputs, layout_or_out):
             if _is_int8_gemm(inputs):
                 # No need to reorder for int8 gemm
+                if input_indices != list(range(len(inputs))):
+                    inp_idx = input_indices[6]
+                    x2_idx = input_indices[7]
+                    # Move bias at front of x2
+                    return [
+                        *[inputs[idx] for idx in input_indices[:6]],
+                        inputs[inp_idx],
+                        inputs[x2_idx],
+                    ], layout_or_out
```

Comment:
Can we always set the `input_indices` in a way to make the input order as [inp, x, w] and reorder it to [x, w, inp] for int8 too? What is the problem if we can reuse the logic of fp32/bf16/fp16?

Reply:
Fixed in https://github.com/pytorch/pytorch/pull/129470